### PR TITLE
fix: [SRM-10232]: Update new logs UI for services page

### PR DESCRIPTION
**Pull request by praneshg239**
##### Screenshots:

Logs tab:
image

Metrics tab:
image

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by praneshg239** [comment]
retrigger jest

**Comment by praneshg239** [comment]
trigger cypress

**Comment by praneshg239** [comment]
trigger cypress

File: src/modules/85-cv/pages/monitored-service/components/ServiceHealth/components/HealthScoreChart/HealthScoreChart.tsx [hunk 1]
```diff
@@ -61,6 +61,12 @@ export default function HealthScoreChart(props: HealthScoreChartProps): JSX.Elem
 
   return (
     <Container className={css.main}>
+      {/* <Layout.Horizontal>
```
Comment (harshilbits) [inline src/modules/85-cv/pages/monitored-service/components/ServiceHealth/components/HealthScoreChart/HealthScoreChart.tsx:64]: Why have we commented this code ?Shall we remove it ?

**Review by harshilbits — APPROVED** [review]

**Review by harshilbits — APPROVED** [review]

**Comment by praneshg239** [comment]
retrigger jest

**Comment by praneshg239** [comment]
retrigger jest

**Pull request merged**